nvflare/private/fed/client/client_runner.py

```diff
@@ -193,10 +204,21 @@ def _process_task(self, task: TaskAssignment, fl_ctx: FLContext) -> Shareable:
         if filter_list:
```

Comment:
I see our filters code (pre VS post) have a lot of duplicates, we should re-factor them in next PR.